node/how_to_with_npm/prisma.md

```diff
@@ -0,0 +1,253 @@
+# How to create a RESTful API with Prisma and Oak
+
+[Prisma](https://prisma.io) has been one of our top requested modules to work
+with in Deno. The demand is understandable, given that Prisma's developer
+experience is top notch and plays well with so many persistent data storage
+technologies.
+
+We're excited to show you how to use Prisma with Deno.
+
+In this How To guide, we'll setup a simple RESTful API in Deno using Oak and
+Prisma.
+
+Let's get started.
+
+1. Setup the application
+
+```shell
+mkdir rest-api-with-prisma-oak
+cd rest-api-with-prisma-oak
+deno run -A npm:prisma init
+```
+
+Update the `prisma/schema.prisma` with the following:
+
+```ts
+generator client {
+  provider = "prisma-client-js"
+  previewFeatures = ["deno"]
+  output = "../generated/client"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Dinosaur {
+  id          Int     @id @default(autoincrement())
+  name        String  @unique
+  description String
+}
+```
+
+Create a `.env` file and update the connection string with your actual
+PostgreSQL connection string.
+
+2. Create the database schema
+
+```shell
+deno run -A npm:prisma db push
+```
+
+3. Generate a Prisma client for Data Proxy
+
+```shell
+deno run -A npm:prisma generate --data-prxoxy
+```
+
+4. Create a repository and connect to GitHub
+
+In order to use Prisma Data Platform, we'll first have to create a GitHub repo.
+
+Initialize your repository, add the remote origin, then push to the repo.
+
+5. Import your Project into Prisma Data Platform
+
+Next, sign up for a free
+[Prisma Data Platform account](https://cloud.prisma.io/).
+
+Click **New Project** and select **Import a Prisma Repository**.
+
+It'll ask for your PostgreSQL connection string, which you have in your `.env`.
+Paste it here. Then click **Create Project**.
+
+You'll receive a new connection string that begins with `prisma://`. You'll use
+it in the next section.
+
+6. Set the Data Proxy Connection string in your environment.
+
+Replace the PostgreSQL connection string in your `.env` file with your new Data
+Proxy connection string.
+
+Now install `dotenv-cli` locally:
+
+```shell
+npm install dotenv-cli
```

Comment:
npm install is necessary? I wonder if https://deno.land/std@0.163.0/dotenv could be used or `deno run -A npm:dotenv-cli` instead?